homeassistant/components/spider.py

```diff
@@ -19,13 +20,15 @@
 DOMAIN = 'spider'
 
 SPIDER_COMPONENTS = [
-    'climate'
+    'climate',
+    'switch'
 ]
 
 CONFIG_SCHEMA = vol.Schema({
     DOMAIN: vol.Schema({
         vol.Required(CONF_PASSWORD): cv.string,
         vol.Required(CONF_USERNAME): cv.string,
+        vol.Optional(CONF_SCAN_INTERVAL, default=120): cv.positive_int,
```

Comment:
This doesn't seem to be used.

Reply:
Yeah I was wondering how I can use it for both components. I couldn't find a good example I believe. Should I save it to a constant and then use @Throttle()? Or what is a good existing example?

Reply:
Nevermind. Think I found a way. Will push the changes tonight

Reply:
Api is now taking care of the refreshing.